components/brave_wallet/common/brave_wallet_types.h

```diff
@@ -9,12 +9,15 @@
 #include <string>
 #include <vector>
 
+#include "brave/third_party/boost/multiprecision/include/boost/multiprecision/cpp_int.hpp"
```

Comment:
this should just be `boost/multiprecision/cpp_int.hpp`

Reply:
Fixed in https://github.com/brave/brave-core/pull/12384/commits/22b9f168985e26718c269284146375e133481fcc